cmake/modules/FindGig.cmake

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
There's a blank line here. Also, do you want to add a copyright notice to this file?

Reply:
I wasn't sure what copyright to put since it's nearly a direct copy of some others.

Reply:
I'm no lawyer, but I imagine something like this should be fine:
```cmake
# Copyright (c) <year> <name>
# Based on <file>, copyright (c) <name> <year>
#
# Redistribution and use is allowed according to the terms of the New BSD license.
# For details see the accompanying COPYING-CMAKE-SCRIPTS file.
```
Besides, if you, the author, are not sure about the copyright, some random person in the future wanting to use this in their project is going to have absolutely no clue.